css-view-transitions-1/Overview.bs

```diff
@@ -1538,6 +1538,14 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 								backdrop-filter: <var>backdropFilter</var>;
 							}
 						}
+
+						@media (prefers-reduced-motion) {
```

Comment:
Is there a WPT test for this? Asking b/c I don't know offhand if the user preference media features can be tested in WPT. If not, merging this PR may be dependent on that test-ability requirement.

Reply:
It's not currently testable. See https://github.com/web-platform-tests/wpt/issues/14839